getting-started/index.html

```diff
@@ -262,14 +262,14 @@ <h3>Find your RCT2 installation</h3>
                             </ol>
                         </div>
                         <div id="content-linux-ubuntu" class="page-distro">
-                            <h2>Install OpenRCT2 for Debian 8, Ubuntu 16.04, 16.10 &amp; Mint 18</h2>
+                            <h2>Install OpenRCT2 for Debian, Ubuntu 18.04, 18.10 &amp; Mint 19</h2>
```

Comment:
I only know about status for Ubuntu. Supported versions for now are 18.04 LTS and whatever is current non-LTS.

Reply:
Mint 19 is based on Ubuntu 18.04.